apps/server/src/modules/fwu/controller/fwu.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get, Req, Res, InternalServerErrorException } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { Authenticate } from '@src/modules/authentication/decorator/auth.decorator';
+import { Request, Response } from 'express';
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { FwuUc } from '../uc/fwu.uc';
+
+@ApiTags('FWU')
+@Authenticate('jwt')
+@Controller('fwu')
+export class FwuController {
+	constructor(private readonly fwuUc: FwuUc) {}
+
+	@Get('*')
+	async get(@Req() req: Request, @Res() res: Response) {
```

Comment:
how about not using the request object, but define the expected param and use class-validator for it?

Reply:
A path parameter 'fwuLearningContent' with corresponding class-validator was added.